GUI/Main.Designer.cs

```diff
@@ -513,10 +509,10 @@ private void InitializeComponent()
             // 
             // Update
             // 
-            this.UpdateCol.HeaderText = "Update";
-            this.UpdateCol.Name = "Update";
-            this.UpdateCol.SortMode = System.Windows.Forms.DataGridViewColumnSortMode.Programmatic;
-            this.UpdateCol.Width = 46;
+            this.Update.HeaderText = "Update";
+            this.Update.Name = "Update";
+            this.Update.SortMode = System.Windows.Forms.DataGridViewColumnSortMode.Programmatic;
+            this.Update.Width = 46;
             // 
```

Comment:
Please revert, or expand name to UpdateColumn or UpdateSelection or some such. "Update" gives us build warnings.